front_end/src/components/layout/Navbar.js

```diff
@@ -106,20 +108,16 @@ class Navbar extends Component {
                 Abode
               </Typography>
             </Link>
-            <Button variant="text" color="inherit">
-              <Link to={{pathname: "/", state: { scroll: "topDeals"}}} style={{color: "white"}}>
+            <AnchorLink href="#topDealsAnchor" offset="-450" style={{color: "white"}}>
+              <Button variant="text" color="inherit">
                 Top Deals
-              </Link>
-            </Button>
-            <Button
-              className={classes.beforeSeperation}
-              variant="text"
-              color="inherit"
-            >
-              <Link to={{pathname: "/", state: { scroll: "featuredCities"}}} style={{color: "white"}}>
+              </Button>
+            </AnchorLink>
+            <AnchorLink href="#featuredCitiesAnchor" offset="-500" className={classes.beforeSeperation}>
+              <Button variant="text" color="inherit">
```

Comment:
Featured Cities turns blue on hover... might be an easy fix to add:
style={{color: "white"}}, like for Top Deals